src/views/partials/header1.hbs

```diff
@@ -0,0 +1,5 @@
+<header>
```

Comment:
Why you have two header files?

Reply:
i needed a small part from the main header to the other page :3
I can replace it if you want, should I do it?

Reply:
Yes, it would be better if you have one header file and call it in all handlebars files, and you can put the main in the home page

Reply:
done @shahenazmonia